src/utils/forwardAuthList.ts

```diff
@@ -0,0 +1,8 @@
+export const ForwardAuthAllowlist = [
+  'Remote-User',
+  'Remote-Email',
+  'Remote-Name',
+  'Cf-Access-Authenticated-User-Email',
+  'X-authentik-username',
```

Comment:
Use camel case like the other headers. It's still not mandatory, but let's stay consistent.

Reply:
These headers are defaults and come from Authentik, Authelia, Cloudflare.

Reply:
Yes, but headers are not case-sensitive. See [RFC 7230, Section 3.2](https://datatracker.ietf.org/doc/html/rfc7230#section-3.2), and you're using them in a function with lowercase.

Reply:
That is very good point! Thank you. 

I'll convert them all to lower case since we are doing that anyway and make it a bit faster